### PR TITLE
adding important note for about using ReFS for CSV

### PR DESCRIPTION
**Pull request by Karl-WE**
Ticket: 120071725000684, 120060924006013 

further references: 
https://social.technet.microsoft.com/Forums/windows/en-US/6b2dcc4f-e735-4700-81f3-df45d94e7e01/refs-for-a-hyperv-csv-volume
https://www.windowspro.de/marcel-kueppers/refs-ntfs-vor-nachteile-dateisysteme-server-2016
Support Tickets: 120071725000684, 120060924006013

### CODE REVIEW

**Comment by Karl-WE** [comment]
Hi @eross-msft any news / suggestions regarding this PR and #4859 (they belong together contextually)

**Comment by eross-msft** [comment]
@JohnMarlin-MSFT  Would you be able to review the failover cluster portion of this? I'm not sure who would be the person for the refs piece, would you know? Thanks!

File: WindowsServerDocs/failover-clustering/failover-cluster-csvs.md [hunk 1]
```diff
@@ -74,6 +74,14 @@ The server uses one of the following I/O redirection modes, depending on the sit
 
 In Windows Server 2012 R2, you can view the state of a CSV volume on a per node basis. For example, you can see whether I/O is direct or redirected, or whether the CSV volume is unavailable. If a CSV volume is in I/O redirected mode, you can also view the reason. Use the Windows PowerShell cmdlet **Get-ClusterSharedVolumeState** to view this information.
 
+> [!IMPORTANT]
+> * Please understand, that CSV's that are pre-formatted with **ReFS, used on top of SAN's will NOT use Direct I/O**, no matter if all other requirements for Direct I/O are met.
```
Comment (illfated) [inline WindowsServerDocs/failover-clustering/failover-cluster-csvs.md:78]: minor English grammar improvement:
```suggestion
> * Please note that CSVs pre-formatted with **ReFS used on top of SANs will NOT use Direct I/O**, regardless of all other requirements for Direct I/O being met.
```

File: WindowsServerDocs/failover-clustering/failover-cluster-csvs.md [hunk 2]
```diff
@@ -74,6 +74,14 @@ The server uses one of the following I/O redirection modes, depending on the sit
...
+> * If you plan to use CSV in junction with SAN(-FrontEnd) attached disks, format drives with NTFS before converting them to a CSV to leverage the performance benefits of Direct I/O.
+> * This behaviour is by design. Please consult links in chapter "More information".
```
Comment (illfated) [inline WindowsServerDocs/failover-clustering/failover-cluster-csvs.md:80]: readability improvement suggestion:
```suggestion
> * This behaviour is by design. Please consult the pages linked in the section **More information** below.
```

**Comment by PRMerger12** [comment]
@Karl-WE : Thanks for your contribution! The author(s) have been notified to review your proposed change. @JasonGerend

**Comment by Karl-WE** [comment]
Thank you Trond for your support and review @illfated it is always helpful to re-read things and refine.

**Comment by illfated** [comment]
You're welcome. I hope this will help to get the review process moved forward, even though I don't have the proper knowledge about this exact topic.

**Review by illfated — APPROVED** [review]
- [X] **Normal source file view:** https://github.com/MicrosoftDocs/windowsserverdocs/pull/4860/files?short_path=0e0583a
- [X] **Rich diff view:** https://github.com/MicrosoftDocs/windowsserverdocs/pull/4860/files?short_path=0e0583a#diff-0e0583aa68603a0640699dce73bfa4f10a528e9ab0e01d0c7e317543fcdbd8b0 (for better readability of the new content)

Looks OK to me, from a grammatical viewpoint.

**Comment by Karl-WE** [comment]
I would like to add that in the named support requests there was a statement from employee that is very deep into the topic but he said he cannot update his old technet > techcommunity articles at the moment to reflect the situation. This also motived my, next to trapping into this performance pitfall, to create this PR.

The fact that we have had 2 different and independent cases on the same issue both (after months of back and forth) ended with the now in this PR captured conclusion.

**Comment by illfated** [comment]
Sounds almost incredible, but at least you have used it for a very constructive purpose. Good to see improvements from contributors like you.

**Comment by eross-msft** [comment]
Confirmed with product team, this is ready to publish.

**Comment by Karl-WE** [comment]
Thank you @eross-msft for the quick double-check

**Pull request merged**